specifications/ietf-eat-profile/cddl/ietf_eat_ocp_profile.cddl

```diff
@@ -1,24 +1,57 @@
 cwt-eat = {
+  ; Mandatory Claims

```

Comment:
Since we will want deterministic encoding of the map keys in ascending order, probably each claim should have (Mandatory) or (Optional) by it.